### PR TITLE
RIA-4927 logging for AIP appealReferenceNumber

### PR DESCRIPTION
**Pull request by bradsorour**
### JIRA link (if applicable) ###

[RIA-4927](https://tools.hmcts.net/jira/browse/RIA-4927)

### Change description ###

Logging appealReferenceNumber to match to ccd case id.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by burale — DISMISSED** [review]
LGTM

**Review by burale — DISMISSED** [review]
LGTM

**Review by gavinslilley — DISMISSED** [review]

**Review by gavinslilley — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealReferenceNumberCcdHandlerHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,129 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit;
```
Comment (rajeshthuraiyur) [inline src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealReferenceNumberCcdHandlerHandlerTest.java:1]: I see the file name as HandlerHandlerTest? Can this be changed to HandlerTest?

**Review by rajeshthuraiyur — DISMISSED** [review]
LGTM

File: src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealReferenceNumberCcdHandlerHandlerTest.java (hunk 1 above)
Comment (bradsorour) [reply src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/AppealReferenceNumberCcdHandlerHandlerTest.java:1]: Yes, good spot @rajeshthuraiyur - the existing name makes no sense.

**Review by rajeshthuraiyur — APPROVED** [review]
LGTM

**Review by augustio — APPROVED** [review]

**Pull request merged**